src/containers/task-group.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Task from '../components/task';
+
+class TaskGroup extends Component {
+
+  processTaskList(list, givenMonth, givenType) {
+    let resultsArray = [];
+    const loop = list.map( (item) => {
+      if(item.month === givenMonth && item.type === givenType) {
+        resultsArray.push(item);
+      };
+    });
+    if(resultsArray.length) {
+      return true;
+    }
+  }
+
+  taskTypeIsUsed(list, givenMonth, givenType) {
```

Comment:
Have refactored but kept 'list' as I found that changing the different 'lists' to specific list names made it less easy for me to understand as the person writing it XD